io.github.wxmaxima_developers.wxMaxima.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.wxmaxima_developers.wxMaxima
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: wxmaxima
+rename-appdata-file: wxMaxima.appdata.xml
+rename-desktop-file: wxMaxima.desktop
+rename-icon: wxmaxima
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Network access
+  # for update checker
```

Comment:
Unless it can autoupdate itself, it seems pointless.

Reply:
No, I think it shouldn't have access to network unless it breaks said built-in function in a way visible to user.

Reply:
> No, I think it shouldn't have access to network unless it breaks said built-in function in a way visible to user.

`Help` → `Check for Updates`